test/test.dart

```diff
@@ -112,12 +112,15 @@ void xtest(String? name, dynamic Function() testFunction) {
 
 Future<void> setupTests(List<String>? args) async {
   parseTestNameFromArguments(args);
+  
+  setUpAll(() async {
+    await setupBaas();
+  });
 
-  await setupBaas();
-  setUp(() {
+  setUp(() async {
```

Comment:
```suggestion
  setUp(() {
```

Reply:
Yes, I have added it probably while I was trying to call `await setupBaas()` in setUp